nirodents/workflows/brainextraction.py

```diff
@@ -456,3 +460,12 @@ def _bspline_distance(in_file, spacings=(8, 2, 8)):
     extent = (np.array(img.shape[:3]) - 1) * img.header.get_zooms()[:3]
     retval = [f"{v}" for v in np.ceil(extent / np.array(spacings)).astype(int)]
     return f"-b {'x'.join(retval)}"
+
+def _lap_sigma(in_file):
+    import numpy as np
+    import nibabel as nb
+    import math
+
+    img = nb.load(in_file)
+    min_vox = np.amin(img.header.get_zooms())
+    return str(0.3508 * math.exp(1.4652 * min_vox))
```

Comment:
This equation is based roughly on creating the desired sigma values for 0.1, 0.5 and 1.0 voxel sizes as 0.4, 0.75, and 1.5 respectively. These values were plot in excel and an exponential trendline of best-fit was used to create the equation here.

As it's only 3 points, it might not be the best-fitting line, so it might be something we need to come back to with different use cases, but it works with some of the data I've tested it on this morning.